templates/common/root/_Gruntfile.js

```diff
@@ -280,6 +284,33 @@ module.exports = function (grunt) {
           sourcemap: true
         }
       }
+    },<% } else { %>
+
+    // Compiles Sass to CSS and generates necessary files if requested
+    sass: {
+      options: {
+        includePaths: [
```

Comment:
Intellij Idea kept wanting to make my import statements look like 
```sass
@import "styles/scss_filename";
```
So rather than fight it, I changed includePaths to
```javascript
includePaths: [
  'bower_components',
  'app'
]
```
in my Grunt file so if Idea ever changed the path when renaming a sass file I wouldn't need to worry about it